draw.io-api/pom.xml

```diff
@@ -88,6 +96,113 @@
             <id>download-draw.io-sources</id>
             <phase>generate-resources</phase>
           </execution>
+          <execution>
+            <id>download-draw.io-gliffy-sources</id>
+            <phase>generate-resources</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <executions>
+          <!-- Move sources to be able to exclude specific files. This can be removed when
+            https://packagecloud.io/jgraph/drawio will contain gae-stub-1.0-3, which is needed for building these
+            files. -->
+          <execution>
+            <id>copy-sources</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${draw.io.path}/src/main/java/copy</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${draw.io.path}/src/main/java</directory>
+                  <excludes>
+                    <exclude>com/mxgraph/online/AbsAuthServlet.java</exclude>
+                    <exclude>com/mxgraph/online/MSGraphAuthServlet.java</exclude>
+                    <exclude>com/mxgraph/online/GoogleAuthServlet.java</exclude>
+                  </excludes>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <!-- Move sources needed for the gliffy import to be able to use the functionality in newer versions of
+          draw.io that no longer contains it. -->
```

Comment:
You do this in 4 executions below. Are you sure it can't be done with a single execution? If that is the case when a comment explaining the limitation might be good.